src/sentry/digests/notifications.py

```diff
@@ -91,41 +100,43 @@ def attach_state(project, groups, rules, event_counts, user_counts):
 
 
 class Pipeline:
-    def __init__(self):
-        self.operations = []
+    def __init__(self) -> None:
+        self.operations: List[Callable[..., Any]] = []
 
-    def __call__(self, sequence):
-        return reduce(lambda x, operation: operation(x), self.operations, sequence)
+    def __call__(self, sequence: Sequence[Any]) -> Any:
+        return reduce(lambda x, operation: operation(x), self.operations, sequence)  # type: ignore
```

Comment:
What was the issue?

Reply:
I didn't know how to add annotations to a lambda. I'll fix this now.